projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Boost_Cache_Settings.php

```diff
@@ -19,7 +19,7 @@ class Boost_Cache_Settings {
 	 * An uninitialized config holds these settings.
 	 */
 	private $default_settings = array(
-		'enabled'    => true,
+		'enabled'    => false,
 		'exceptions' => array(),
 		'logging'    => false,
```

Comment:
Can you also adjust this while you are at it. Too small of a change for a separate PR. We are saving them  as `bypass_patterns` but the default says `exceptions`.
```suggestion
		'enabled'         => false,
		`bypass_patterns' => [],
		'logging'         => false,
```